egs/aishell/ASR/conformer_ctc/decode.py

```diff
@@ -229,18 +287,41 @@ def decode_one_batch(
         subsampling_factor=params.subsampling_factor,
     )
 
+    if params.method == "ctc-decoding":
+        best_path = one_best_decoding(
+            lattice=lattice, use_double_scores=params.use_double_scores
+        )
+        # Note: `best_path.aux_labels` contains token IDs, not word IDs
+        # since we are using H, not HLG here.
+        #
+        # token_ids is a lit-of-list of IDs
+        token_ids = get_texts(best_path)
+
+        # hyps is a list of str, e.g., ['xxx yyy zzz', ...]
+        hyps = bpe_model.decode(token_ids)
```

Comment:
Did you change the units to BPE tokens?  I did not see the pipeline in the `prepare.sh`.

Reply:
just be compatible for recipe that has Chinese and English  as multi_cn , for AISHELL i use char unit.